core-plugins/docs/SSH-action.md

```diff
@@ -40,9 +40,9 @@ This example runs a script on demo@example.com:
     {
         "name": "SSHAction",
           "plugin": {
-            "name": "SSHAction",
```

Comment:
should we rename the `name` field of the example config.json?
https://github.com/caskdata/hydrator-plugins/blob/bd71d29fbbc6699c21f9aa9716c7d197fa6201d9/core-plugins/docs/SSH-action.md#L41-L41